extensions/amp-story/1.0/amp-story-system-layer.js

```diff
@@ -354,9 +402,35 @@ export class SystemLayer {
    */
   onMutedStateUpdate_(isMuted) {
     this.vsync_.mutate(() => {
-      isMuted ?
-        this.getShadowRoot().setAttribute(AUDIO_MUTED_ATTRIBUTE, '') :
-        this.getShadowRoot().removeAttribute(AUDIO_MUTED_ATTRIBUTE);
+      if (isMuted) {
+        this.getShadowRoot().setAttribute(AUDIO_MUTED_ATTRIBUTE, 'muted');
+      } else if (this.storeService_.get(StateProperty.PAGE_HAS_AUDIO_STATE)) {
+        this.getShadowRoot().setAttribute(AUDIO_MUTED_ATTRIBUTE, 'audioon');
+      } else {
+        this.getShadowRoot().setAttribute(AUDIO_MUTED_ATTRIBUTE, 'noaudio');
+      }
+    });
+  }
```

Comment:
This is going to display a message when user clicks to mute/unmute the story.
Product question: do we also want to display this message when the user navigates to the next page?

Reply:
I clarified with Hong earlier and she said that the messages should only appear on the button clicks and not on the next page navigation.